cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split1/FaceSuite.scala

```diff
@@ -217,6 +218,8 @@ class IdentifyFacesSuite extends TransformerFuzzing[IdentifyFaces] with Cognitiv
     .collect()
     .map(r => r.getString(0)).toSeq
 
+  IdentifyFacesSuite.cleanOldGroups()
+
```

Comment:
No need to make a companion object, and please call this in afterAll

Reply:
Done